test/test_testing.py

```diff
@@ -4073,6 +4073,228 @@ def test_add_layer_merge(self, client: _TestingPebbleClient):
             == plan.to_yaml()
         )
 
+    def test_add_layer_checks_combine_override_replace(self, client: _TestingPebbleClient):
+        plan = client.get_plan()
+        assert isinstance(plan, pebble.Plan)
+        assert plan.to_yaml() == '{}\n'
+        client.add_layer(
+            'foo',
+            pebble.Layer("""\
+            checks:
+                up:
+                    override: replace
+                    level: alive
+                    period: 30s
+                    threshold: 1
+                    exec:
+                        command: service nginx status
+            """),
+        )
+        plan = client.get_plan()
```

Comment:
It probably doesn't add much value to do this intermediate `get_plan()` and assertion -- can probably just drop those to shorten the test a bit.

Reply:
Agree :)